start-from-zero/Javascript/src/views/index/index.js

```diff
@@ -5,19 +5,68 @@ function redirectToEvaluation(){
         e.stopPropagation();
         e.preventDefault();
         app.innerHTML = EvaluationsPage();
-        redirectToLogin();
+        
         redirectToNewEvaluation();
     });
 }
 
 function redirectToNewEvaluation(){
+    app.innerHTML = EvaluationsPage();
     const newEvaluations = document.querySelector('.new-evaluations-page');
     newEvaluations.addEventListener('click',function(e){
         e.stopPropagation();
         e.preventDefault();
         app.innerHTML = NewEvaluationPageAll();
-        redirectToLogin();
+        
         redirectToEvaluation();
+
+        const btn = document.getElementById('btn');
+        var form = document.getElementById('form-new');
+
+        btn.addEventListener('click',function(){
+            function User(username,password,date,checkbox,textarea,select){
+            this.username = username;
+            this.password = password;
+            this.date = date;
+            this.checkbox = checkbox
+            this.textarea= textarea;
+            this.select = select;
+            }
+
+            const username = document.getElementById('username-id').value;
+            const password = document.getElementById('password-id').value;
+            const calendar = document.getElementById('calendar-id').value;
+            const checkbox = document.querySelectorAll('.input-checkbox');
+            const textarea = document.querySelectorAll('textarea');
+            const select = document.querySelectorAll('select');
+            const option = document.querySelectorAll('option');
+
+            const textareaArr = [];
+            textarea.forEach(element => {
+                textareaArr.push(element.value);
+            })
+
+            const selectArr = [];
+            select.forEach(element => {
+                selectArr.push(element.value);
+            })
+
+            const checkboxArr = [];
+            checkbox.forEach(element => {
+                if(element.checked === true){
+                    checkboxArr.push(element.value + ' ' + element.checked)
+                }else{
+                    checkboxArr.push(element.checked);
+                }
+            })
+            
+            const userObj = new User(username,password,calendar,checkboxArr,textareaArr,selectArr);
```

Comment:
why do you need to store the login credentials alongside the evaluation data? I would think these are 2 different forms that are not coupled in any way;

Reply:
That makes sense. So where I should store the credentials ?

Reply:
the localStorage does not limit you to just one key; you can have a credential key there to store the login data;